pytest_plugins/manual_skipped.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import pytest
+
+from robottelo.constants import NOT_IMPLEMENTED
+
+LOGGER = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items, config):
+    """
+    Collects tests based on pytest option to select tests marked as failed on Report Portal
+    """
+    manualskip_args = config.getvalue('mark_manuals_skipped')
+    if not manualskip_args:
```

Comment:
Just suggestion, Can we do it this way?
```
if config.getvalue('mark_manuals_skipped'):
   for item in items:
        if item.get_closest_marker(name='stubbed'):
           item.add_marker(marker=pytest.mark.skip(reason=NOT_IMPLEMENTED))
else:
     return
```

Reply:
I think both its the same and the suggestion won't be helping much in any manner.

But the current implementation quickly exits out of code if the marker is not mentioned in the pytest. Putting else loop unnecessarily increasing the conditioning.

But still I am buying your removal of stubbed var idea.